admin/class-option-tabs-formatter.php

```diff
@@ -27,7 +27,12 @@ public function run( WPSEO_Option_Tabs $option_tabs, Yoast_Form $yform, $options
 
 		echo '<h2 class="nav-tab-wrapper" id="wpseo-tabs">';
 		foreach ( $option_tabs->get_tabs() as $tab ) {
-			printf( '<a class="nav-tab" id="%1$s-tab" href="#top#%1$s">%2$s</a>', $tab->get_name(), $tab->get_label() );
+			printf(
+				'<a class="nav-tab" id="%1$s" href="%2$s">%3$s</a>',
+				esc_attr( $tab->get_name() . '-tab' ),
+				esc_url( '#top#' . $tab->get_name() ),
+				$tab->get_label()
```

Comment:
Was there a specific reason not to use `esc_html` on the `$tab->get_label()` ?
It seems these are translated strings that just contain a word in the implementations.

Reply:
> Was there a specific reason not to use esc_html on the $tab->get_label() ?

These PRs are all part of a larger change to escape all output. These PRs are created iteratively and bits are pulled in separate PRs.

The initial subset which was pulled mainly focused on escaping I18n function calls.
This second subset pulled focuses mainly on correct attribute escaping.
A next iteration will be tracing back some of the more complex ones, such as this `$tab->get_label()` call, verifying that expected string type and determining what type of output escaping would be most appropriate for those calls.

> It seems these are translated strings that just contain a word in the implementations.